samcli/commands/_utils/options.py

```diff
@@ -362,7 +362,7 @@ def notification_arns_override_option(f):
 
 
 def _space_separated_list_func_type(value):
-    return value.split(" ") if not isinstance(value, tuple) else value
+    return value.split(" ") if not isinstance(value, tuple) and not isinstance(value, list) else value
```

Comment:
Hi @G4brym, thank you for your contribution!

I think this code could be clearer:
```python
    if isinstance(value, str):
        return value.split(" ")
    if isinstance(value, (list, tuple)):
        return value
    raise ValueError()
```

What do you think?